citrination_client/search/dataset/tests/test_dataset_query.py

```diff
@@ -38,6 +38,14 @@ def test_dataset_search(self):
                     id=Filter(equal='151278')))))
         assert 1 == response.total_num_hits
 
+    def test_search_limit_enforced_dataset_search(self):
```

Comment:
Here, if you had the MAX_QUERY_RESULTS defined as a constant, you could just import that instead of using a hard coded value here.

Reply:
@joannehill seconded.